File: app/src/main/java/com/gadgetsfolk/kidsgk/activity/UpdateVersionActivity.java
package com.gadgetsfolk.kidsgk.activity;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import com.gadgetsfolk.kidsgk.R;
import com.google.android.gms.ads.AdRequest;
import com.google.android.gms.ads.AdView;
import butterknife.BindView;
import butterknife.ButterKnife;
import butterknife.OnClick;

public class UpdateVersionActivity extends AppCompatActivity {
    @BindView(R.id.adView)
    AdView adView;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_update_version);
        ButterKnife.bind(this);

        AdRequest adRequest = new AdRequest.Builder().build();
        adView.loadAd(adRequest);
    }

    @OnClick(R.id.update_now_button)
    void updateNow(){
        Intent intent = new Intent("android.intent.action.VIEW", Uri.parse("https://play.google.com/store/apps/details?id=com.gadgetsfolk.kidsgk"));
        startActivity(intent);
    }
}

File: app/src/main/java/com/gadgetsfolk/kidsgk/activity/UpdateVersionActivity.kt
package com.gadgetsfolk.kidsgk.activity

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.gadgetsfolk.kidsgk.databinding.ActivityUpdateVersionBinding
import com.google.android.gms.ads.AdRequest

class UpdateVersionActivity : AppCompatActivity() {
    private lateinit var binding:ActivityUpdateVersionBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityUpdateVersionBinding.inflate(layoutInflater)
        val view = binding.root
        setContentView(view)

        val adRequest = AdRequest.Builder().build()
        binding.adView.loadAd(adRequest)

        binding.updateNowButton.setOnClickListener{
            updateNow()
        }
    }

    fun updateNow() {
        val intent = Intent("android.intent.action.VIEW", Uri.parse("https://play.google.com/store/apps/details?id=com.gadgetsfolk.kidsgk"))
        startActivity(intent)
    }

}